### PR TITLE
Adds wklink

### PR DESCRIPTION
**Pull request by normanrz**

### CODE REVIEW

File: wklink/links.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+4871:https://demo.webknossos.org/annotations/Explorational/5b55dbaa1c0000bd008c0e80/readOnly#5149,653,57,0,2.42
```
Comment (jstriebel) [inline wklink/links.txt]: I guess this is just for testing purposes, and we keep track of the links in our infrastructure, right?

File: wklink/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (jstriebel) [inline wklink/LICENSE]: Should we just use an MIT License for the whole repo? Or might this create conflicts with the software we use in the image?

File: wklink/links.txt (hunk 1 above)
Comment (normanrz) [reply wklink/links.txt]: Yeah, I'll put in different links for testing. The real stuff is in kube

File: wklink/LICENSE (hunk 2 above)
Comment (normanrz) [reply wklink/LICENSE]: Good question. We can put the dockerfiles + config under MIT, but keep the original license of the software

Comment (normanrz) [reply wklink/LICENSE]: Not sure how to word that, tho. Let's delay to a different PR.

Comment (normanrz) [reply wklink/LICENSE]: I moved the license up. Helm does it the same way. They also have a separate license for the charts.

**Comment by jstriebel** [comment]
:+1:

**Pull request merged**